src/components/Navigation/Menu/InnerMenu.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import BackButton from "../Buttons/BackButton";
+import classesCss from "./Menu.module.scss"
+
+export default function InnerMenu({children, onBackHandler, navigationClasses, menuClasses}) {
+
+    const navigationClassesWrap = [classesCss.NavigationBlock];
+    navigationClassesWrap.push(navigationClasses)
+
+    const menuClassesWrap = [classesCss.MenuModule];
+    menuClassesWrap.push(menuClasses)
+
+    return(
+        <div className={menuClassesWrap.join(" ")}>
```

Comment:
Why not to just write those classes there?
There's also popular lib for classes called `classnames`, please have a look.